ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/CollectionSelection.tsx

```diff
@@ -104,7 +105,13 @@ function CollectionSelection({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [data]);
 
-    function onToggleCollectionModal() {
+    function onOpenViewCollectionModal() {
+        setModalAction({ type: 'view', collectionId: selectedScopeId });
+        setIsCollectionModalOpen((current) => !current);
+    }
+
+    function onOpenCreateCollectionModal() {
+        setModalAction({ type: 'create' });
```

Comment:
When trying to create a collection, we will change the `modalAction` accordingly